### PR TITLE
revise word2vec according to new code

### PR DESCRIPTION
**Pull request by helinwang**

### CODE REVIEW

File: word2vec/README.md [hunk 1]
```diff
@@ -165,109 +165,24 @@ CBOW的好处是对上下文词语的分布在词向量上进行了平滑，去
...
-        'dictfile': 'data/vocabulary.txt'}
-		
-define_py_data_sources2(
-    train_list="data/train.list",
-    test_list="data/test.list",
-    module="dataprovider",
-    obj="process",
-    args=args)
-```
+本章训练的是5-Gram，表示在PaddlePaddle训练时，每条数据的前4个词用来预测第5个词。PaddlePaddle提供了对应PTB数据集的python包`paddle.dataset.imikolov`，自动做数据的下载与预处理，方便大家使用。
```
Comment (wangkuiyi) [inline word2vec/README.md]: 5-Gram => 5-gram 模型

File: word2vec/README.md [hunk 2]
```diff
@@ -165,109 +165,24 @@ CBOW的好处是对上下文词语的分布在词向量上进行了平滑，去
...
+```text
+<s> I have a dream
+I have a dream that
+have a dream that one
+a dream that one day
+dream that one day <e>
 ```
 
-### 模型结构
+## 配置说明
```
Comment (wangkuiyi) [inline word2vec/README.md]: 配置说明 => 编程实现 ？

**Review by wangkuiyi — APPROVED** [review]
with minor change comments

File: word2vec/README.md (hunk 1 above)
Comment (helinwang) [reply word2vec/README.md]: Done.

File: word2vec/README.md (hunk 2 above)
Comment (helinwang) [reply word2vec/README.md]: Done.

**Pull request merged**